day-trader-java-app/daytrader-gateway/src/main/java/com/ofss/daytrader/gateway/controller/GatewayController.java

```diff
@@ -737,7 +761,152 @@
+		}
+	}
+
+	public ResponseEntity<QuoteDataBean> getQuoteFallback(@PathVariable("symbol") String symbol) {
+		String quotekey = "getQuote_" + symbol;
+
+		if (CachedObjectsClass.getInstance().checkCacheForObject(quotekey) != null) {
+			System.out.println("data is displayed from cache");
+			QuoteDataBean quoteDataBean = (QuoteDataBean) CachedObjectsClass.getInstance()
+					.checkCacheForObject(quotekey);
+			return new ResponseEntity<QuoteDataBean>(quoteDataBean, getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+			QuoteDataBean quoteData = new QuoteDataBean("s:701", "Company701", 9.00, new BigDecimal(195),
+					new BigDecimal(195), new BigDecimal(1), new BigDecimal(10), 1.0);
+
+			return new ResponseEntity<QuoteDataBean>(quoteData, getNoCacheHeaders(), HttpStatus.OK);
+		}
+	}
+
+	public ResponseEntity<Collection<OrderDataBean>> getOrdersFallback(@PathVariable("userId") String userId) {
+		Collection<OrderDataBean> orders = null;
+		if (CachedObjectsClass.getInstance().checkCacheForObject(userId) != null) {
+			System.out.println("data is displayed from cache");
+			QuoteDataBean quoteDataBean = (QuoteDataBean) CachedObjectsClass.getInstance().checkCacheForObject(userId);
+			return new ResponseEntity<Collection<OrderDataBean>>(orders, getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+
+			return new ResponseEntity<Collection<OrderDataBean>>(orders, getNoCacheHeaders(), HttpStatus.OK);
+		}
+
+	}
+
+	public ResponseEntity<AccountDataBean> getAccountDataFallback(@PathVariable("userId") String userId) {
+		String DataKey = "AccountData" + userId;
+		AccountDataBean accountData = new AccountDataBean();
+		if (CachedObjectsClass.getInstance().checkCacheForObject(DataKey) != null) {
+			System.out.println("data is displayed from cache");
+			accountData = (AccountDataBean) CachedObjectsClass.getInstance().checkCacheForObject(DataKey);
+			return new ResponseEntity<AccountDataBean>(accountData, getNoCacheHeaders(), HttpStatus.OK);
+
+		} else {
+			accountData.setAccountID(1);
+			accountData.setProfileID(userId);
+			accountData.setBalance(new BigDecimal(-1));
+			accountData.setOpenBalance(new BigDecimal(-1));
+			return new ResponseEntity<AccountDataBean>(accountData, getNoCacheHeaders(), HttpStatus.OK);
+
+		}
+	}
+
+	public ResponseEntity<AccountProfileDataBean> getAccountProfileDataFallback(@PathVariable("userId") String userId) {
+		String profileDataKey = "AccountProfileData" + userId;
+
+		AccountProfileDataBean accountProfileData = new AccountProfileDataBean();
+		if (CachedObjectsClass.getInstance().checkCacheForObject(profileDataKey) != null) {
+			System.out.println("data is displayed from cache");
+			accountProfileData = (AccountProfileDataBean) CachedObjectsClass.getInstance()
+					.checkCacheForObject(profileDataKey);
+			return new ResponseEntity<AccountProfileDataBean>(accountProfileData, getNoCacheHeaders(), HttpStatus.OK);
```

Comment:
this line and another same line in this method, move it to the end of the method, so that the line appears only once

Reply:
changes are done as mentioned